source/core/ut_suite_manager.pkb

```diff
@@ -376,12 +383,16 @@ create or replace package body ut_suite_manager is
     else
       l_path := lower( a_path );
     end if;
-
-    open l_result for
+    l_suite_item_name := case when l_tag_list is not null then 'suite_items_tags' else 'suite_items' end;
+    
+    
+    
+    /* Rewrite that as tags should be put on whats left not on full suite item cache */
+    l_sql :=
```

Comment:
It might be good to have a chat on that query. I'd like to understand how it works after changes.

Reply:
sure

Reply:
The query I've created is quite a monster itself.
Do you think it would be beneficial if it is changed to be a template, like we did in cursor_compare?

Reply:
Definitely I will do that